### PR TITLE
stdlib, resources: fixed style issue in isa.hh

### LINKED ISSUE #147: Fix style in RISC-V isa file
https://github.com/gem5/gem5/blob/dceabe5fda598937b00959cc648fb7f2fbbe950a/src/arch/riscv/isa.hh#L73C19-L73C19

This should be `rvType` instead of `rv_type`.

### PR DESCRIPTION
**Pull request by Harshil2107**
Changed "rv_type" to "rvType".

Change-Id: I7432a87d7a37324777385707854aefba2475b98c

### CODE REVIEW

File: src/arch/riscv/isa.hh [hunk 1]
```diff
@@ -70,7 +70,7 @@ enum FPUStatus
 class ISA : public BaseISA
 {
   protected:
-    RiscvType rv_type;
```
Comment (rogerchang23424) [inline src/arch/riscv/isa.hh:75]: It has the public `rvType` for public function. Perhaps it should be `_rvType`?

**Review by rogerchang23424 — COMMENTED** [review]
Thanks for code style fix.

File: src/arch/riscv/isa.hh (hunk 1 above)
Comment (powerjg) [reply src/arch/riscv/isa.hh:75]: +1 I think this could cause a build issue?

Comment (Harshil2107) [reply src/arch/riscv/isa.hh:75]: I changed all occurrences to _rvType and just changing that line was also causing errors. The errors are in isa.cc is it ok to change rv_type to _rvType there as well?

**Comment by powerjg** [comment]
I hate to ask, but could you squash these two commits into one?

**Comment by Harshil2107** [comment]
I think I squashed them, please let me know if they are not.

**Review by powerjg — APPROVED** [review]

**Review by rogerchang23424 — APPROVED** [review]

**Pull request merged**